sklearnex/tests/test_memory_usage.py

```diff
@@ -226,15 +227,20 @@ def _kfold_function_template(estimator, dataframe, data_shape, queue=None, func=
     else:
         name = estimator.__name__
 
+    threshold = (
+        EXTRA_MEMORY_THRESHOLD_PANDAS
+        if isinstance(dataframe, pd.DataFrame)
```

Comment:
could it be `pd.Series` somehow?

Reply:
Thanks for pointing this out. It made me realize that I was actually checking the wrong parameter ("X" here would be the dataframe, whereas this "dataframe" is a string) - fixed it now.